annot8-components-easyocr/src/main/java/io/annot8/components/easyocr/processors/InitSettings.java

```diff
@@ -0,0 +1,31 @@
+/* Annot8 (annot8.io) - Licensed under Apache-2.0. */
+package io.annot8.components.easyocr.processors;
+
+public class InitSettings {
+
+  public static final boolean DEFAULT_DOWNLOAD = false;
+  public static final boolean DEFAULT_GPU = false;
+  public static final String DEFAULT_LANGS = "en";
+
```

Comment:
Are these defaults required if you must explicitly set values below? From a quick look, I couldn't see where (if anywhere) they were used.

Reply:
yep - not used anymore, defaults are now in the main settings